packages/strapi-design-system/.eslintrc

```diff
@@ -0,0 +1,34 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020, // Allows for the parsing of modern ECMAScript features
+    "sourceType": "module", // Allows for the use of imports
+    "ecmaFeatures": {
+      "jsx": true // Allows for the parsing of JSX
+    }
+  },
+  "settings": {
+    "react": {
+      "version": "detect" // Tells eslint-plugin-react to automatically detect the version of React to use
+    }
+  },
+  "extends": [
+    "plugin:react/recommended", // Uses the recommended rules from @eslint-plugin-react
+    "plugin:prettier/recommended" // Enables eslint-plugin-prettier and eslint-config-prettier. This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
+  ],
+  "rules": {
+    "react/sort-prop-types": 1,
+    "default-case": "error",
+    "no-unused-vars": ["error", { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }],
+    "no-console": ["error", { "allow": ["warn", "error"] }]
+  },
+  "overrides": [
+    {
+      "files": "*.mdx",
+      "parser": "eslint-mdx",
+      "rules": {
+        "react/react-in-jsx-scope": "off",
+        "no-console": 0
+      }
+    }
+  ]
+}
```

Comment:
It would be nice if we can extend the [@strapi/esling-config](https://github.com/strapi/eslint-config/blob/main/front.js). I think it would be better to have a more consistent front-end codebase between all the Strapi projects.

Reply:
No need to do that move in your right now PR btw.

Reply:
Agree - that's a nice task for another PR!

Reply:
Yeah i looked into that previously. But agreed, it would be easier to do that.